application/controllers/RoleController.php

```diff
@@ -244,6 +246,9 @@ public function auditAction()
             ]
         ));
 
+        $this->view->title = $title;
```

Comment:
Please use `setTitle` as suggested.

```suggestion
        $this->setTitle($title);
```